index.html

```diff
@@ -9,11 +9,26 @@
     />
     <title>AllStarAuto</title>
     <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
apparently these preconnect links optimizes it so that it knows that it should fetch from these urls soon